apps/codesearch/certificate.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: networking.gke.io/v1
+kind: ManagedCertificate
+metadata:
+  name: codesearch
+  namespace: codesearch
+  labels:
+    app: codesearch
+spec:
+  domains:
+  - cs.k8s.io
```

```suggestion
  - cs-canary.k8s.io
```

The certificate will fail to be issued if the ip address don't match the load balancer.